core/domain/stats_jobs_one_off.py

```diff
@@ -1581,3 +1581,72 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class ResetExplorationIssuesOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job to reset all ExplorationIssues to a valid/empty state."""
```

Comment:
Is this a job we will only ever run once then remove, or will it be run multiple times? If the former, please define why this is needed (e.g. the context & investigation that led to this solution) & what it will fix. If the latter, please specify when we should run it (e.g. what validation jobs determine the need for running this).